x-pack/plugin/watcher/build.gradle

```diff
@@ -86,6 +87,10 @@ test {
   systemProperty 'es.set.netty.runtime.available.processors', 'false'
 }
 
+internalClusterTest {
+  systemProperty 'es.set.netty.runtime.available.processors', 'false'
```

Comment:
Same, I suspect this is _only_ needed for integration tests.

Reply:
indeed: https://github.com/elastic/elasticsearch/pull/61802/commits/1f4ca9a0ea0276a9c23b9ec69b4ad0e1f6a01bf7